conda_lock/conda_lock.py

```diff
@@ -129,6 +130,14 @@
     KIND_LOCK: "conda-lock install --name YOURENV {lockfile}",
 }
 
+_implicit_cuda_message = """
+  'cudatoolkit' package added implicitly without specifying that cuda packages
+  should be accepted.
+  Add a minimum cuda version via `--with-cuda VERSION` or via virtual packages
```

Comment:
If I'm not mistaken, the version is not "minimum" but represents the exact version of CUDA which is available on the (hypothetical) target system.

Reply:
Yes, it's the version used for the solve, but in practice cuda versions are treated as lower bounds. So setting a version in the hypothetical solve is effectively setting the minimum version for which the locked environment can be used. But I switched it to 'specify' to avoid trying to say too much.